playbooks/prometheus_agent.yml

```diff
@@ -6,16 +6,22 @@
     wdir: "{{ base_work_directory | default('/var/lib/osp-observability') }}"
     container_name: "{{ prometheus_container_name | default('prometheus_agent') }}"
     container_image: "{{ prometheus_image | default('quay.io/prometheus/prometheus:latest') }}"
-    remote_write: "{{ prometheus_remote_write }}"
+    remote_write: "{{ prometheus_remote_write }}"                        
   become: true
   become_method: sudo
   tasks:
+    - name: Include scrape endpoints file
+      ansible.builtin.include_vars:
+        file: /var/lib/osp-observability/scrape
```

Comment:
Are you sure the path is correct? The default in observabilityclient can is "~/.osp-observability/scrape-endpoints.yaml" as can be seen in [1]. Also it would be great if the path won't be hardcoded. Having some var defined similarly as are those above in this playbook and possibility to override it from outside of this playbook by (let's say) "prometheus_scrape_config" would be ideal.

[1] https://github.com/infrawatch/python-observabilityclient/pull/1/files#diff-f818275d2c8a182c29ce25e2d8ad0d60029e85c872ec1f5a2cc4c08a9f985b7cR43

Reply:
I updated it to use the default "~/.osp-observability/scrape-endpoints.yaml" to be more sure